symbols/qitmeer/coinbase/coinbase12.go

```diff
@@ -0,0 +1,42 @@
+package coinbase
+
+import (
+	`github.com/Qitmeer/qitmeer-miner/common`
+	`github.com/Qitmeer/qitmeer/core/address`
+	`github.com/Qitmeer/qitmeer/core/types`
+)
+
+type Coinbase12 struct {
+	CoinbaseBase
+}
+
+func (this *Coinbase12) GetCoinbaseTx() *types.Tx {
```

Comment:
There is a lot of duplicated code comparing with the `coinbase.go`, better to eliminate them.

Reply:
update